docs/_posts/2021-09-17-seminar-minutes-2021-06.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Bot Friday 2021年第6次活动纪要
+author: qhduan
+categories: events
+tags:
+  - minutes
+  - blog
+  - wechaty
+header:
+  teaser: /assets/2021/seminar-06/group.jpg
+---
+
+<< Bot5 2021年第5次活动纪要: [Bot5 Season 2021 Episode 5](http://www.bot5.ml/events/seminar-minutes-2021-05/) <<
+
+## Bot Friday Season 2021 Episode 6
+
+- 时间：2021年9月17日晚 6:30 举办形式: 线上 Zoom + 线下
+- 地点：句子互动，北京市海淀区智造大街F座5F
+- 轮值主席：
+    1. Chair: 段清华
+    2. Vice-Chair: 张惟师
+
+## Attendees
+
+1. 王海良，运营 Chatopera 云服务，开发者友好的智能对话机器人平台 **本次分享者**
+1. 段清华，知未智能CTO&Co-founder **本次分享者**
+1. 李卓桓，Wechaty 作者, 基于 Conversational User Interface 的用户认证流程探讨
+1. 孙天祥，Authing Developer **本次分享者**
+1. 李佳芮, 句子互动创始人
+1. 董吉洋 **本次分享者**
+1. vfyang，嘉楠神州
+1. 邹念义，Authing PM
+
+新人:
+
+1. 孙天祥
+1. 邹念义
+
+### 视频
+
+<div class="video-container" style="
+    position: relative;
+    padding-bottom:56.25%;
+    padding-top:30px;
+    height:0;
+    overflow:hidden;
+">
+<iframe width="560" height="315" src="https://www.youtube.com/embed/a8snyCoNAfQ" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Please add the issue link for the video description in youtube.

Reply:
added